src/posix/platform/radio.cpp

```diff
@@ -151,7 +151,7 @@ void Radio::Init(void)
         {
             power = static_cast<int8_t>(strtol(str, nullptr, 0));
             error = sRadioSpinel.SetChannelMaxTransmitPower(channel, power);
-            if (error != OT_ERROR_NONE && error != OT_ERROR_NOT_FOUND)
+            if (error != OT_ERROR_NONE && error != OT_ERROR_NOT_IMPLEMENTED)
```

Comment:
Log warning if not implemented?

Reply:
If the user sets the parameter `max-power-table`, it means the user hopes to send the max power table to the RCP. If the RCP doesn't support the max power table, the host die directly. It makes the user find the issue early. So I think it is better to die rather than output warning here. Thoughts?

Reply:
If the host die directly, it breaks the compatibility between the host and RCP. So it is better to output a warning log than die directly when the command is not implemented. Will add a warning log.

Reply:
Updated